.travis.yml

```diff
@@ -22,7 +23,8 @@ env:
 
 matrix:
   include:
-    - env: TOXENV=translations SQLALCHEMY_DATABASE_URI="$SQLALCHEMY_DATABASE_TEST_URI" PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/TrueNTH-USA-site-config/develop/site_persistence_file.json"
+    - env: TOXENV=translations PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/TrueNTH-USA-site-config/develop/site_persistence_file.json"
+    - env: TOXENV=translations PERSISTENCE_FILE="https://raw.githubusercontent.com/uwcirg/ePROMs-site-config/develop/site_persistence_file.json"
```

Comment:
Aren't you simply masking the TrueNTH value of PERSISTENCE_FILE when resetting it for ePROMs?